keep.tex

```diff
@@ -343,7 +343,7 @@ \subsection{sMPC and the blockchain}
 The advent of the blockchain enables secure usage of sMPC in
 adversarial scenarios. By using a public blockchain as an immutable
 ledger, sMPC can be made secure in the face of a dishonest
-supermajority \cite{spdz}, and with the requirement of a network
+supermajority \cite{spdz}, and, with the requirement of a network
```

Comment:
+1 on the comma fix, but... would this be best split into two sentences?

Reply:
I could see that, too. Can make that tweak if you'd like.

Reply:
Let's wait, and do it in another PR if it's still an issue- we've got a bunch of changes in the pipeline re: system design

Reply:
:100: